.github/workflows/sync.yml

```diff
@@ -0,0 +1,52 @@
+name: Sync
+
+on:
+  push:
+    branches: [master, staging]
+  workflow_dispatch:
+
+jobs:
+  sync-master:
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/master'
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 15
+      - name: Install awscli
+        run: pip install --user awscli
```

Comment:
Looks like the container may already have this installed https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md#cli-tools.

You probably don't need to install it. I wonder if it's already added to the path. You might be able to do `aws` instead of `~/.local/bin/aws`.

Reply:
Yeah, when I was researching I found two different examples, one where they installed and the other they didn't. I went with this first because it was closer to what we currently had. But I'm all for simplifying where we can so I'll remove the install step and try using just ```aws```.